.github/checkgroup.yml

```diff
@@ -32,23 +32,23 @@ subprojects:
       - ".actions/**"
     checks:
       # Note: updates here should be applied to the lightning_lite group
-      - "pl-cpu (ubuntu-20.04, 3.7, 1.9, oldest)"
-      - "pl-cpu (ubuntu-20.04, 3.7, 1.12)"
-      - "pl-cpu (ubuntu-20.04, 3.8, 1.9)"
-      - "pl-cpu (ubuntu-20.04, 3.8, 1.10)"
-      - "pl-cpu (ubuntu-20.04, 3.9, 1.11)"
-      - "pl-cpu (ubuntu-20.04, 3.9, 1.12)"
-      - "pl-cpu (ubuntu-20.04, 3.10, 1.12)"
-      - "pl-cpu (macos-11, 3.7, 1.9, oldest)"
-      - "pl-cpu (macos-11, 3.7, 1.12)"
-      - "pl-cpu (macos-11, 3.8, 1.10)"
-      - "pl-cpu (macos-11, 3.9, 1.11)"
-      - "pl-cpu (macos-11, 3.10, 1.12)"
-      - "pl-cpu (windows-2022, 3.7, 1.9, oldest)"
-      - "pl-cpu (windows-2022, 3.7, 1.12)"
-      - "pl-cpu (windows-2022, 3.8, 1.10)"
-      - "pl-cpu (windows-2022, 3.9, 1.11)"
-      - "pl-cpu (windows-2022, 3.10, 1.12)"
+      - "pl-cpu (macos-11, pytorch, 3.7, 1.9, oldest)"
+      - "pl-cpu (macos-11, pytorch, 3.7, 1.12)"
+      - "pl-cpu (macos-11, pytorch, 3.8, 1.10)"
+      - "pl-cpu (macos-11, pytorch, 3.9, 1.11)"
+      - "pl-cpu (macos-11, pytorch, 3.10, 1.12)"
+      - "pl-cpu (ubuntu-20.04, pytorch, 3.7, 1.9, oldest)"
+      - "pl-cpu (ubuntu-20.04, pytorch, 3.7, 1.12)"
+      - "pl-cpu (ubuntu-20.04, pytorch, 3.8, 1.9)"
+      - "pl-cpu (ubuntu-20.04, pytorch, 3.8, 1.10)"
```

Comment:
Diff will look nicer if you keep the previous order of having ubuntu first.

Reply:
just wanted to be consistent as other has MAc-Ubuntu-Win